Source/Core/DolphinQt2/Settings/PathPane.cpp

```diff
@@ -127,20 +127,27 @@ QGroupBox* PathPane::MakeGameFolderBox()
 
   m_remove_path->setEnabled(false);
 
-  auto* checkbox = new QCheckBox(tr("Search Subfolders"));
-  checkbox->setChecked(SConfig::GetInstance().m_RecursiveISOFolder);
+  auto* recursive_checkbox = new QCheckBox(tr("Search Subfolders"));
+  recursive_checkbox->setChecked(SConfig::GetInstance().m_RecursiveISOFolder);
+
+  auto* auto_checkbox = new QCheckBox(tr("Check for Gamelist changes in the background"));
```

Comment:
Game list is supposed to be two words.

Also, the capitalization is inconsistent. If you want to follow how we normally capitalize things, I think it would be "Check for Game List Changes in the Background".